config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -88,6 +88,14 @@ data:
       - "Throttling: Rate exceeded"
       installFailingReason: AWSAPIRateLimitExceeded
       installFailingMessage: AWS API rate limit exceeded
+    # This issue is caused by AWS throttling the CreateHostedZone request. The terraform provider is not properly
+    # handling the throttling response and gets stuck in a state where it does not retry the request. Eventually,
+    # the terraform provider times out claiming that it is waiting for the hosted zone to be INSYNC.
+      - name: AWSRoute53Timeout
```

Comment:
Unindent this. And don't forget to rerun `make update`.

Reply:
yikes. done.